source/connection_manager.c

```diff
@@ -726,37 +730,14 @@ static void s_final_destruction_task(struct aws_task *task, void *arg, enum aws_
     struct aws_http_connection_manager *manager = arg;
```

Comment:
comment a few lines up needs to change "If there's no cull task we just destroy the
* manager directly without a cross-thread task" isn't true anymore